spec/21-http_response_header_format.md

```diff
@@ -1,113 +1,107 @@
-# Trace Context HTTP Response Headers Format
+# Trace Context Server Timing Metric Format
 
-This section describes the binding of the distributed trace context to the `traceresponse` HTTP header.
+This section describes the binding of the distributed trace context to a metric in the Server Timing HTTP header.
 
-## Traceresponse Header
+## Trace Context Metric
 
-The `traceresponse` HTTP response header field identifies a completed request in a tracing system. It has four fields:
+The trace context metric identifies a completed request in a tracing system. It has four params:
 
-* `version`
-* `trace-id`
-* `child-id`
-* `trace-flags`
+* `tid` - required
+* `cid` - required
+* `v` - optional
+* `flags` - optional
 
-### Header Name
+Example server timing header with trace context metric:
 
-Header name: `traceresponse`
-
-The header name is [ASCII case-insensitive](https://infra.spec.whatwg.org/#ascii-case-insensitive). That is, `TRACERESPONSE`, `TraceResponse`, and `traceresponse` are considered the same header. The header name is a single word; it does not contain any delimiters such as a hyphen.
+```
+server-timing: trace;tid=0af7651916cd43dd8448eb211c80319c,cid=b7ad6b7169203331
+```
 
-In order to increase interoperability across multiple protocols and encourage successful integration, tracing systems SHOULD encode the header name as [ASCII lowercase](https://infra.spec.whatwg.org/#ascii-lowercase).
+### Metric Name
 
-### traceresponse Header Field Values
+Metric name: `trace`
```

Comment:
As someone who hasn't attended the meetings (where I suppose this was discussed), this is a bit obscure. What does this name mean? Here, it's `trace`, but should it always be that? What would be other good values?

Reply:
server-timing is a set of metrics. This is one of them and we are trying to reserve the name `trace`.

Alternative naming proposal is here: https://github.com/w3c/trace-context/issues/561